server/container/guice/guice-common/src/main/java/org/apache/james/jmap/JMAPCommonModule.java

```diff
@@ -54,6 +56,8 @@ protected void configure() {
 
         bind(MailSpool.class).in(Singleton.class);
         bind(MailFactory.class).in(Singleton.class);
+
+        bind(AutomaticallySentMailDetector.class).to(AutomaticallySentMailDetectorImpl.class);
```

Comment:
No scope?
If you give no scope, as this class doesn't have a constructor, we may not need an interface; then it may be instanciated every time.